docs/develop/integrations/fastly.md

```diff
@@ -0,0 +1,186 @@
+
+Navigate to the [tokens](https://console.gomomento.com/tokens) page, and choose the cloud provider and region you used to create your cache. Since the cache is already created, we will use a fine-grained token that will allow the worker to read from and write to the cache; but will not allow it to do control plane operations such as delete. This is especially helpful if you want to manage the security of control plane and data plane operations separately.
+
+Choose the `Fine-Grained Access Token` token type, select `worker` as `Cache Name` from the drop down, and `readwrite` as `Role Type`. The `Super User Token` is used for managing control plane operations. More information about Momento authentication can be found [here](https://docs.momentohq.com/develop/basics/working-with-momento-auth-tokens). Hit the `Generate Token` button.
+
+![Generate token](/img/fgac-worker-auth.png)
+
+Copy the `Auth Token` and `HTTP Endpoint` and save it in a safe place. You'll need to use it later to configure your Fastly Compute@Edge deployment.
+
+![Generated token](/img/http-endpoint-auth-token.png)
+
+### Fastly Setup
+
+Once you have created your Fastly account, you're ready to configure the Fastly side of things using the [Fastly console](https://manage.fastly.com/services/all). Once you're logged in, click on your account name in the top right corner and select `Account`. Under the `Peronsal profile` section of the side bar, select `API tokens`. Click on the `Create Token` button in the top right and follow Fastly's instructions for creating a [user API token](https://docs.fastly.com/en/guides/using-api-tokens#creating-api-tokens). Make sure to save this API token in a safe place.
+
+Next, you will install the [Fastly CLI](https://developer.fastly.com/learning/compute/#install-the-fastly-cli) and follow the command line prompts for providing the API token you just created:
+
+```bash
+brew install fastly/tap/fastly
+fastly profile create
+```
+
+## Deploying with Fastly
+
+Now it's time to configure and deploy the application to your Fastly account. As noted before, you need a copy of the Momento JavaScript SDK available in your repository. 
+In this example, we'll deploy from a GitHub repository forked from [the original](https://github.com/momentohq/client-sdk-javascript).
+
+![Fork SDK Repository](/img/github-fork-js-sdk.png)
+
+This repository contains a directory under `examples` for `fastly-compute` which contains the example code for interacting with your Momento Cache using the Momento HTTP API. The HTTP API is lightweight in that you won't need any additional dependencies beyond what Fastly requires and you can use the standard `fetch` HTTP client methods. However, it only provides a basic subset of all of the Momento APIs, such as `get`, `set`, and `delete`.
```

Comment:
We should add the info about the HTTP API earlier in the doc before we mention why we only support AWS.